packaging/nnstreamer.spec

```diff
@@ -102,6 +105,22 @@ Requires:	nnstreamer = %{version}-%{release}
 %description tensorflow-lite
 NNStreamer's tensor_fliter subplugin of TensorFlow Lite.
 
+%package python-filter
+Summary:  NNStreamer Python Custom Filter Support
+Requires: nnstreamer = %{version}-%{release}
+Requires: python
+Requires: python-numpy-devel
```

Comment:
You do not need devel package at runtime. You need the main package.

Reply:
Got it